proposal.adoc

```diff
@@ -223,6 +223,10 @@ The `io.vertx` groupId and package name must remain the same.
  
 Docker images and NPM packages should keep the same name too. Their description would be updated to “Eclipse Vert.x”.
 
+=== Versionning
+
+The released artifacts would still use a _major.minor.micro_ version scheme. 
```

Comment:
Not strictly. We don't follow the "bumping" rules.

So far we use something like:

* major: vert.x model version / full rewrite (or close) 
* minor: new features, new components, some breaking changes
* micro: bug fixes, no breaking changes